ggshield/verticals/sca/output/schemas.py

```diff
@@ -1,15 +1,41 @@
-from dataclasses import dataclass
-from typing import cast
+from dataclasses import dataclass, field
+from typing import List, cast
 
 import marshmallow_dataclass
 from pygitguardian.models import BaseSchema
-from pygitguardian.sca_models import SCAScanAllOutput, SCAScanDiffOutput
+from pygitguardian.sca_models import (
+    SCALocationVulnerability,
+    SCAScanAllOutput,
+    SCAScanDiffOutput,
+    SCAVulnerablePackageVersion,
+)
+
+
+@dataclass
+class SCAJSONVulnerablePackageVersion(SCAVulnerablePackageVersion):
```

Comment:
`SCAVulnerablePackageVersion` does not have these attributes, they must be removed from `SCAVulnerability`